### PR TITLE
Glide vendoring

### PR DESCRIPTION
**Pull request by jdavis92**
@mattnenterprise and I want gorets to manage its own vendoring.  Each dependency references the most recent commit at the time of this pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,8 @@ language: go
...
-  - go get golang.org/x/net/context/ctxhttp
-  - go get golang.org/x/net/html/charset
-  - go get golang.org/x/text/encoding
-  - go get golang.org/x/text/transform
+    - export GO15VENDOREXPERIMENT=1
 
 go:
-  - 1.3
-  - 1.4
   - 1.5
```
Comment (kelcecil) [inline .travis.yml]: You should add Go 1.6 to this list.

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,31 @@
-all: test
+all: ci-deps test clean
 
 test:
-	go test -v ./...
+	go test -v $$(go list ./... | grep -v /vendor/)
```
Comment (kelcecil) [inline Makefile]: "glide novendor" might more easily give you what you're after here.

**Comment by kelcecil** [comment]
+1 for this. Two small notes, but LGTM otherwise.

**Comment by mattnenterprise** [comment]
:thumbsup:

**Pull request merged**